src/vector/defn.rs

```diff
@@ -66,4 +74,16 @@ impl<'a> Field<'a> {
         let rv = unsafe { ogr::OGR_Fld_GetNameRef(self.c_field_defn) };
         return _string(rv);
     }
+
+    pub fn get_type(&'a self) -> i32 {
```

Comment:
As this will return a `OGRFieldType` you could change the method name to `field_type`.